
The first half of this is certainly upstream-worthy, and improves the
detection of the include directories for KDE 4, But the second half is
due to some Fedora layout oddness that is not reported by kde4-config,
so it has to be manual on RHELL or Fedora.

			Nico Kadel-Garcia <nkadel@gmail.com>
			2013/06/19      	  

--- subversion-1.8.0/build/ac-macros/kwallet.m4.kwallet	2013-04-10 08:39:13.000000000 -0400
+++ subversion-1.8.0/build/ac-macros/kwallet.m4	2013-06-20 00:06:20.847963361 -0400
@@ -63,15 +63,14 @@
                   fi
                 done
                 qt_include_dirs="`$PKG_CONFIG --cflags-only-I QtCore QtDBus QtGui`"
-                kde_dir="`$KDE4_CONFIG --prefix`"
-                SVN_KWALLET_INCLUDES="$DBUS_CPPFLAGS $qt_include_dirs -I$kde_dir/include"
+                kde_include_dirs="-I`$KDE4_CONFIG --path include`"
+                SVN_KWALLET_INCLUDES="$DBUS_CPPFLAGS $qt_include_dirs $kde_include_dirs"
                 qt_libs_other_options="`$PKG_CONFIG --libs-only-other QtCore QtDBus QtGui`"
                 SVN_KWALLET_LIBS="$DBUS_LIBS -lQtCore -lQtDBus -lQtGui -lkdecore -lkdeui $qt_libs_other_options"
                 CXXFLAGS="$CXXFLAGS $SVN_KWALLET_INCLUDES"
                 LIBS="$LIBS $SVN_KWALLET_LIBS"
                 qt_lib_dirs="`$PKG_CONFIG --libs-only-L QtCore QtDBus QtGui`"
-                kde_lib_suffix="`$KDE4_CONFIG --libsuffix`"
-                LDFLAGS="$old_LDFLAGS `SVN_REMOVE_STANDARD_LIB_DIRS($qt_lib_dirs -L$kde_dir/lib$kde_lib_suffix)`"
+                LDFLAGS="$old_LDFLAGS `SVN_REMOVE_STANDARD_LIB_DIRS($qt_lib_dirs)` -L$libdir/kde4/devel"
                 AC_LANG(C++)
                 AC_LINK_IFELSE([AC_LANG_SOURCE([[
 #include <kwallet.h>
